### PR TITLE
Add genologics recipe

### PR DESCRIPTION
**Pull request by vladsaveliev**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.

Python API for Clarity Genologics LIMS

### CODE REVIEW

File: recipes/genologics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - requests
+
+  run:
+    - python 2.6.*|2.7.*  # [py2k]
```
Comment (bgruening) [inline recipes/genologics/meta.yaml]: can you please remove this here and skip the build for py3k in your build section with `skip: True  # [py3k]`

Comment (vladsaveliev) [inline recipes/genologics/meta.yaml]: The module is supported by python 3. That's why the following 2 lines - I wanted to specify version ranges for both py2 and py3:
```
    - python 2.6.*|2.7.*  # [py2k]
    - python >=3.4  # [py3k]
```
I also tried one line `python 2.6.*|2.7.*|>=3.4`, but it didn't work properly. Is there a standard way to specify python version range more specifically? I didn't find any examples, probably the way to do is just not to specify it? I can just remove the version ranges.

Comment (bgruening) [inline recipes/genologics/meta.yaml]: In BioConda we build automatically against py27,py35, py36, see here: https://github.com/bioconda/bioconda-recipes/blob/bulk/scripts/env_matrix.yml#L1

So this is not needed :)

Comment (vladsaveliev) [inline recipes/genologics/meta.yaml]: So python versions outside of those 3 are not supported anyway?

Comment (bgruening) [inline recipes/genologics/meta.yaml]: No. Is this a problem for you?

Comment (vladsaveliev) [inline recipes/genologics/meta.yaml]: Not at all, removed version ranges :)

Comment (bgruening) [inline recipes/genologics/meta.yaml]: Cool, thanks!

File: recipes/genologics/build.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$PYTHON setup.py install --single-version-externally-managed --root=/
```
Comment (bgruening) [inline recipes/genologics/build.sh]: Sorry, just see this now. Can you please remove the `--root=/`. I don't think this is needed.

Comment (vladsaveliev) [inline recipes/genologics/build.sh]: Removed, thanks!

**Comment by vladsaveliev** [comment]
Thank you for merging and help @bgruening

**Pull request merged**